content/components/page-header.mdx

```diff
@@ -0,0 +1,295 @@
+---
+title: Page header
+reactId: drafts_page_header
+description: Page header determines the top-level headings of a UI.
+figma: https://www.figma.com/file/Ee0OrXuOLXMDqUW83EnDHP/PageHeader-(FY23-Q1)?node-id=0%3A1&t=8cKRbkNYhwbqEoHj-1
+---
+
+import {Box, Text} from '@primer/react'
+import ComponentLayout from '~/src/layouts/component-layout'
+export default ComponentLayout
+
+## Usage
+
+The page header arranges elements that apply context and provide navigation for the current page. 
+
+The elements within Page header are rearranged and displayed accordingly depending on the [viewport size](/foundations/layout#viewport-ranges) and context of the page.
+
+<img 
+    src="https://user-images.githubusercontent.com/586552/222194474-74b72ae3-f8d4-4336-9a2c-9d5c18d91c42.png"
+    role="presentation"
+    width="960"
+    alt="Page header component"
+/>
+
+## Anatomy 
+
+Page header can include a context bar, title bar, description, and contextual navigation (pertaining to the current page). 
+
+By default, the [context bar](#context-bar) is visible on narrow viewports only, the [title bar](#title-bar) is visible on all viewports, and the back button and trailing action are visible on regular viewports.
```

Comment:
The back button is renamed to `LeadingAction` so it should be "...and the leading and trailing action are visible on regular viewports."

Would it be also possible to update this on Figma and take a new screenshot please? 🙇🏻‍♀️

Reply:
Fixed! apologies - you told me that and i thought it was the other way around. Should be fixed to 'leading action' now

Reply:
done ✔️